src/libraries/Common/src/Interop/Windows/NCrypt/Interop.ErrorCode.cs

```diff
@@ -20,6 +20,12 @@ internal enum ErrorCode : int
             NTE_NO_MORE_ITEMS = unchecked((int)0x8009002a),
             E_FAIL = unchecked((int)0x80004005),
             STATUS_UNSUCCESSFUL = unchecked((int)0xC0000001),
+            TPM_E_PCP_BUFFER_TOO_SMALL = unchecked((int)0x80290406),
```

Comment:
nit: the enum appears to be sorted by value (except for E_FAIL)... maybe this should be moved up a line?

Reply:
I value sorted the enum (moved the new one and `E_FAIL`).